config/config-mp/src/test/resources/custom-application.json

```diff
@@ -0,0 +1,10 @@
+{
+  "string": "String",
+  "number": 175,
+  "array": [
+    "One",
+    "Two",
```

Comment:
This file should not be here - config-mp does not depend on hocon

Reply:
Overlooked too so removing this